cranelift/filetests/filetests/runtests/fma.clif

```diff
@@ -1,6 +1,8 @@
+test interpret
 test run
 target aarch64
 target s390x
+target x86_64 has_avx=false has_fma=false
```

Comment:
Can you also check it with fma instructions?

Reply:
Yes! As soon as https://github.com/bytecodealliance/wasmtime/pull/4539 is merged.

Although that needs a rebase.